### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by bitfield**
Many, many more complaints about Go.

### CODE REVIEW

**Comment by dgryski** [comment]
LGTM.

File: entries.json [hunk 1]
```diff
@@ -568,4 +568,38 @@
...
 	]
+},{
+	"URL": "https://bluxte.net/musings/2018/04/10/go-good-bad-ugly/",
+	"Author": "Sylvain Wallez",
+	"Year": 2018,
+	"Complaints": [
+		"no generics",
+		"sort.Interface approach is clumsy",
+		"designers did not consult the literature",
+		"Fisher-Price assembly language",
```
Comment (swallez) [inline entries.json]: LOL! I like the term even if I didn't imply it was simple/bad: just very surprising :-)

**Review by swallez — APPROVED** [review]
Non-binding LTGM, and only for the commit related to my post.

Thanks @bitfield! I wanted to prepare a PR for this repo, but it's even better is someone does it for me :-)

**Comment by bitfield** [comment]
Can this PR be merged, then, @ksimka?

**Pull request closed without merging**